### ISSUE TITLE
Favorites not working Kodi 17.6

### ISSUE THREAD

**Issue by tarvcode** [holisticdioxide/Ultimate-Whitecream#18]
Running Kodi 17.6 on both PC and AndroidTV.  Latest version 1.1.64.  I believe the first version installed was 1.1.61 if that helps.  Issue was always there.

Adding a favorite seems to add it somewhere, as once the stream is favorited the context will then show "remove from favorites".  This remains even after restarting app.

Going into the favorites folder shows no items.

In latest version, going into the settings and trying to backup favorites gives a message "no favorites to back up". 

Clearing the favorites using the settings menu does not seem to help either.  Clearing the settings and then going to a stream that was previously favorited, context menu still shows "remove from favorites" indicating it's still favorited.

**Comment by holisticdioxide** [comment]
Favorites seem to be working for me, can you please provide the exact steps to reproduce the issue?

**Comment by tarvcode** [comment]
It's strange, as it's the same issue on two instances here.  Tried to delete and reinstall the addon.  Both are fresh installs of Kodi 17.6 as well.  I'm really just going to a folder or stream or video and using the context menu to add favorite.

Some thoughts;

One difference is I'm running a SQL database for Video/Music, but I don't think that should effect the Addons DB.

Another difference would be the location of the Addons.db file.  On my Android install I have the internal storage and an SD card for storage, maybe it's mixing up the correct location for the DB file?  If you look on the Kodi Wiki it does state locations are kind of all over the place.

Same thing on the Windows10 install.  I decided to use the Windows Store to install (meh) and instead of using %APPDATA% it uses %LOCALAPPDATA%.

I'm looking at the Kodi log and not seeing any relevant information.
